values.yaml

```diff
@@ -67,7 +71,10 @@ global:
     database: ""
     # -- Port for the external redis
     port: ""
-
+    # -- secret name for username and password. assumes secret contains keys: "CONVOY_REDIS_USERNAME", "CONVOY_REDIS_PASSWORD"
+    secret:
+      enabled: false
```

Comment:
If this is set to false by default, what value would be used for the corresponding keys?